db/migrations/20200403185552_add_zomato_id_to_merchants.js

```diff
@@ -0,0 +1,9 @@
+
+exports.up = knex => knex.schema.table('merchants', (table) => {
+  table.integer('zomato_id').unsigned().index();
+});
+
+exports.down = knex => knex.schema.table('merchants', (table) => {
```

Comment:
So in knex, `knex migrate:down` is usually the reverse operation of `knex migrate:up` in order to "rollback" the migration.

I think you meant to do: `table.dropColumn('zomato_id');` here?

You would actually put the:
```
  table.dropColumn('postal_code');
  table.dropColumn('address');
```

In the `exports.up`, but in this case, there's no harm in just keeping the columns as well right?

Reply:
got it